src/map/Map.cpp

```diff
@@ -161,12 +162,31 @@ void Map::genWalls(const vector<MapPoint>& wallStart) {
  * Function Interface: void mapLoadToGame()
  * Description:
  * Creates the wall structures using the GameManager createWall function.
+ * Creates and loads shop and dropzone positions.
  */
 void Map::mapLoadToGame() {
+    // Random shop position variable.
+    int pos;
+    // Random number generator.
+    mt19937 ran;
+    // Shop position being loaded.
+    MapPoint shopPosition;
+
     for (const auto& w : walls) {
-        logv("Creating wall");
         GameManager::instance()->createWall(w.x, w.y, w.width, w.height);
     }
+
+    // Random number generation.
+    ran.seed(1);
```

Comment:
There is a default seed used in the constructor
No need to manually seed it unless you want to guarantee that the shop and drop zones will be identical across every run by using the same seed